cli/lib/api.js

```diff
@@ -0,0 +1,21 @@
+const fetch = require('node-fetch');
+
+class Api {
```

Comment:
Feel free to discard this suggestion if it is not useful, but have you thought about copy-pasting the internal storage client? The external and internal urls are _pretty similar_ and it might be a way to get a head start.

Reply:
I could for sure. I was worried that it might be overwhelming to review as well as require a lot of rebasing since there's a lot of code moving around right now. What do you think?

Reply:
Go for what's easiest for now. The more I think about it, it's probably unlikely that it'd be helpful to duplicate the code, especially if there's a tricky rebase involved. Where there are API methods that are common to the internal and external clients we can make sure to keep those method names aligned as we add them, too.